node-runner-cli/config/CoreDockerConfig.py

```diff
@@ -44,7 +46,12 @@ def ask_data_directory(self):
         if os.environ.get(MOUNT_LEDGER_VOLUME, "true").lower() == "false":
             self.data_directory = None
         if self.data_directory:
-            Path(self.data_directory).mkdir(parents=True, exist_ok=True)
```

Comment:
How does this change anything? The method already allows "exist_ok". Can we have a unit test that shows how this is not working and why we do need to catch the FileExistsError? That will also be relevant for any regression testing.

Reply:
well I've added this because of the error here: https://github.com/radixdlt/babylon-nodecli/actions/runs/6317636456/job/17155013000. I'm not sure why mkdir raises that error but it did in CI even though we had the exist_ok flag set to true